sanitize_test.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
was this file intentionally be added?

Reply:
This was private file, somehow sneaked into public !
Removed it now